enrollment-server-onboarding/pom.xml

```diff
@@ -299,6 +299,43 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>embedded-db</id>
```

Comment:
I would rather use another profile id such as `liquibase` but I am consistent with `wultra/mobile-utility-server`.

Reply:
The `liquibase` option seems better to me.